starters/cms/src/data.ts

```diff
@@ -174,7 +248,12 @@ export async function syncExistingCollection(
             return { didSync: false }
         }
 
-        await syncCollection(collection, dataSource, existingFields, slugField)
+        await syncCollection(
+            collection,
+            dataSource,
+            existingFields as ManagedCollectionFieldInput[],
+            slugField as ManagedCollectionFieldInput
```

Comment:
Why do you need to cast the types here? Casts are dangerous because you are overriding something which might not be true

Reply:
I've removed the typecasting for `slugField`. But I struggle with `existingFields`, I might need some help since those types are not documented.